draft-ietf-cbor-serialization.md

```diff
@@ -127,20 +129,21 @@ This full set was not explicitly named in {{-cbor}}.
 General serialization consists of all of these:
 
 * Any length CBOR argument (e.g., the integer 0 may be encoded as 0x00, 0x1800 or or 0x190000 and so on).
-* Any length floating point regardless of value (e.g. 0.00 can be 0xf900, 0xfa000000000 and so on).
+* Any length floating point value regardless of encoding size (e.g. 0.00 can be 0xf900, 0xfa000000000 and so on).
 * Both definite or indefinite-length strings, arrays and maps are allowed.
 * Big numbers can represent values that are also representable by major types 0 and 1 (e.g., 0 can be encoded as a big number, as 0xc34100).
 
 A decoder that supports general serialization is able to decode all of these.
 
 If a CBOR-based protocol specification does not explicitly specify serialization, general serialization is implied.
 This means that a compliant decoder for such a protocol is required to accept all forms allowed by general serialization including both definite and indefinite lengths.
-For example, CBOR Web Token, {{-CWT}} does not specify serialization; therefore, a full and proper CWT decoder must be able to handle variable-length CBOR argments plus indefinite-length strings, arrays and maps.
+For example, CBOR Web Token (CWT), {{-CWT}} does not specify serialization; therefore, a full and proper CWT decoder must be able to handle variable-length CBOR arguments plus indefinite-length strings, arrays and maps.
+
+> Note that CBOR Object Signing and Encryption (COSE) {{-COSE}} uses effectively ordinary encoding (see {{OrdinarySerialization}}) for its internal structures, such as the `Sig_structure`.
```

Comment:
Section 9 of RFC 9052 references section 4.2.1 of RFC 8949 which is about deterministic serialization. In principle COSE, absolutely requires determinism for the three structures.

Typically, there are no maps in these structures. When there are no maps ordinary == deterministic. However there can be maps in the values of parameters. For example the 'crit' header parameter is a map.

I don't think COSE should be mentioned in the context of ordinary serialization at all.